kivymd/theming.py

```diff
@@ -40,7 +40,6 @@
 
 from kivymd import images_path
 from kivymd.color_definitions import colors, hue, palette
-from kivymd.font_definitions import theme_font_styles  # NOQA: F401
```

Comment:
It cannot be deleted (Flake8).

Reply:
Does `theme_text_color` used in any other files?
like `from kivymd.theming import theme_font_styles`

Reply:
If after the import the `# NOQA: F401` tag is set, then it is used and this import cannot be deleted.